features/FxA-54-push-on-status-change/README.md

```diff
@@ -0,0 +1,84 @@
+# Push Notifications for Account Status Changes
+
+https://mozilla.aha.io/features/FXA-54
+
+## Stories
+
+As a user signed in to an FxA connected device,
+I want my device to update its UI in a timely manner
+when I interact with my account from other devices.
+
+As a developer of client code for a connected device,
+I want to easily receive notifications so that I can
+respond to account status changes without resorting to
+polling the server.
+
+As a developer of Firefox Accounts,
+I want to stop Firefox from polling for account verification
+because this is the single biggest source
+of system load in production.
```

Comment:
from mtg: Ordering of device registration in the desktop patch. Desktop needs to register with auth server BEFORE verification.  cc @philbooth, we need to check this.